### PR TITLE
Update base64 to 0.21

### PR DESCRIPTION
**Pull request by citreae535**
## PR Type

Dependency update

## PR Checklist

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt.
- [x] (Team) Label with affected crates and semver status.

## Overview

Update base64 to 0.21 (MSRV 1.57) and refactor deprecated API calls

### CODE REVIEW

File: actix-http-test/CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased - 2022-xx-xx
 - Minimum supported Rust version (MSRV) is now 1.59.
+- Update `base64` to `0.21`.
```
Comment (robjtede) [inline actix-http-test/CHANGES.md]: as these are not user facing changes, no need for any changelog updates 👍🏻

File: awc/src/builder.rs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 use std::{convert::TryFrom, fmt, net::IpAddr, rc::Rc, time::Duration};
 
+use base64::{prelude::BASE64_STANDARD, Engine};
```
Comment (robjtede) [inline awc/src/builder.rs]: if prelude is being used, then lets just use it

```suggestion
use base64::prelude::*;
```

**Review by robjtede — CHANGES_REQUESTED** [review]

**Comment by citreae535** [comment]
A cursory search shows that `base64` does not seem to be used by `actix-http-test`. Maybe it can be removed from that crate.

**Review by robjtede — APPROVED** [review]
thanks 👍🏻

**Pull request merged**